packages/audits/src/helpers/getLastUpdatedTime.ts

```diff
@@ -0,0 +1,22 @@
+import moment from 'moment';
+
+export const getLastUpdatedTime = (date: moment.MomentInput, currentDate = moment()) => {
+  const seconds = currentDate.diff(date, 'seconds');
```

Comment:
you can check the TimeAgo. see in team-table how i calculate the timeAgo in joined date.